src/app/components/participants-dialog/participants-dialog.component.html

```diff
@@ -2,7 +2,7 @@
     <div id="title"><h3>Participants</h3></div>
     <div id="close">
         <mat-icon class="close-icon" (click)="closeModal()">
-            Close
+            close
```

Comment:
Sentence case

Reply:
the icon is close, if i rename it to Close the X icon stops displaying